File: app/src/main/java/com/vsa/filmoteca/data/repository/ws/FilmotecaInterface.java
package com.vsa.filmoteca.data.repository.ws;


import retrofit2.http.GET;
import retrofit2.http.Url;
import rx.Observable;

/**
 * Created by albertovecinasanchez on 8/3/16.
 */
public interface FilmotecaInterface {

    @GET("/es/webs-municipales/filmoteca/agenda/folder_listing")
    Observable<String> moviesListHtml();

    @GET
    Observable<String> movieDetail(@Url String url);

}

File: app/src/main/java/com/vsa/filmoteca/data/repository/ws/FilmotecaInterface.kt
package com.vsa.filmoteca.data.repository.ws


import retrofit2.http.GET
import retrofit2.http.Url
import rx.Observable

/**
 * Created by albertovecinasanchez on 8/3/16.
 */
interface FilmotecaInterface {

    @GET("/es/webs-municipales/filmoteca/agenda/folder_listing")
    fun moviesListHtml(): Observable<String>

    @GET
    fun movieDetail(@Url url: String): Observable<String>

}
